app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -1155,7 +1155,7 @@ public void onClick(DialogInterface dialog, int whichButton) {
                 return new AlertDialog.Builder(this)
                         .setTitle(R.string.confirm_discard_draft_message_title)
                         .setMessage(R.string.confirm_discard_draft_message)
-                        .setPositiveButton(R.string.cancel_action, new DialogInterface.OnClickListener() {
+                        .setPositiveButton(com.fsck.k9.ui.base.R.string.cancel_action, new DialogInterface.OnClickListener() {
```

Comment:
Import as `import com.fsck.k9.ui.base.R as BaseR`

Reply:
Unfortunately, that's not possible because it's a `.java` file.